lua/nvim-treesitter/health.lua

```diff
@@ -128,7 +128,7 @@ function M.check()
           table.insert(error_collection, { parser_name, query_group, err })
         end
       end
-      table.insert(parser_installation, out)
+      table.insert(parser_installation, vim.trim(out, " ", 2))
```

Comment:
why the additional parameters ? I dont see them in the documentation

Reply:
ooops, meant to use `vim.fn.trim` which is right-trim only